fedimg/services/ec2/ec2initiate.py

```diff
@@ -91,7 +91,8 @@ def main(image_urls, access_id, secret_key, regions, volume_types=None,
         try:
             source = get_source_from_image(image_url)
             if not source:
-                raise ValueError
+                _log.debug('Source not found.')
+                return []
```

Comment:
this looks good for now but in the future I hope to get more output in the logs other than `Source not found.`. I opened https://github.com/fedora-infra/fedimg/issues/118 for this

Reply:
Thanks!